Virtual-Coffee/virtualcoffee.io
# [Feature] Created  sitemap

## Linked Issue
closes #1170 


## Description

This pull request introduces a new `generate-sitemap.mjs` file to automatically generate and update the sitemap for the website. The script consolidates the sitemap generation logic and ensures it runs after the build process.



## Methodology

**New Script:** Created generate-sitemap.mjs to generate the sitemap.
**Postbuild Hook:** Updated package.json to include a `postbuild` script that runs the sitemap generation script automatically after the build process.


## Verification
- Run Build: 
``` bash
pnpm run build
```
- Generate Sitemap: 
``` bash
pnpm run generate-sitemap
```
- Check Output: Verify `sitemap.xml` is generated correctly.
## Code of Conduct

> By submitting this pull request, you agree to follow our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/)
